### PR TITLE
[3.x] Backport AudioStream icons from 4.0

### PR DESCRIPTION
**Pull request by DeeJayLSP**
I just think it's kind of bland without them.

Yes, I changed their colors to match 3.x colors.

~~(I believe I'm supposed to reformat them, I didn't know Inkscape would add info and format it)~~ Done.

### CODE REVIEW

File: editor/icons/icon_audio_stream_microphone.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><linearGradient id="a" gradientUnits="userSpaceOnUse" x1="8" x2="8" y1="1" y2="15"><stop offset="0" stop-color="#ff7a7a"/><stop offset=".5" stop-color="#e1dc7a"/><stop offset="1" stop-color="#66ff9e"/></linearGradient><path d="m7 1c-1.108 0-2 .892-2 2h2v1h-2v2h2v1h-2c0 1.108.892 2 2 2v4l-2 2h6l-2-2v-4c1.108 0 2-.892 2-2h-2v-1h2v-2h-2v-1h2c0-1.108-.892-2-2-2z" fill="url(#a)"/></svg>
```
Comment (YuriSizov) [inline editor/icons/icon_audio_stream_microphone.svg:1]:
```suggestion
<svg height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><linearGradient id="a" gradientUnits="userSpaceOnUse" x1="8" x2="8" y1="1" y2="15"><stop offset="0" stop-color="#ff7a7a"/><stop offset=".5" stop-color="#e1dc7a"/><stop offset="1" stop-color="#66ff9e"/></linearGradient><path d="m7 1c-1.108 0-2 .892-2 2h2v1h-2v2h2v1h-2c0 1.108.892 2 2 2v4l-2 2h6l-2-2v-4c1.108 0 2-.892 2-2h-2v-1h2v-2h-2v-1h2c0-1.108-.892-2-2-2z" fill="url(#a)"/></svg>
```

**Comment by YuriSizov** [comment]
Seems like you've used a different account (an email associated with a different account) to force-push this PR, so GitHub treats it as if you were a first-time contributor. It's not critical, but you might want to check your settings.

**Review by Calinou — APPROVED** [review]
Tested locally, it works as expected:

image

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**